README.md

```diff
@@ -7,6 +7,7 @@ An Ansible role for installing [Apache Spark](https://spark.apache.org).
 - `spark_version` - Spark version.
 - `spark_cloudera_distribution` - Cloudera distribution version (default: `cdh5.4`)
 - `spark_env_extras` - An optional dictionary with key and value attributes to add to `spark-env.sh` (e.g. `MESOS_NATIVE_LIBRARY: "/usr/local/lib/libmesos.so"`)
+- `spark_defaults_extras` - An optional list (not a dictionary, to keep given ordering) with settings to `spark-defaults.conf` (e.g. `"spark.eventLog.enabled   true"`)
```

Comment:
exactly!
The ordering is the main reason, but also with my change we can easily write a config file, where the keys and value are nicely aligned vertically (by adding the right number of spaces between the key and value)